luigi_pipeline/lib/hail_tasks.py

```diff
@@ -48,6 +48,10 @@ class HailMatrixTableTask(luigi.Task):
     genome_version = luigi.Parameter(description='Reference Genome Version (37 or 38)')
     vep_runner = luigi.ChoiceParameter(choices=['VEP', 'DUMMY'], default='VEP', description='Choice of which vep runner'
                                                                                             'to annotate vep.')
+    ignore_missing_samples = luigi.BoolParameter(default=False, description="Allow missing samples in the callset.")
+    skip_sample_subset = luigi.BoolParameter(default=False, description=
+        "Skip subsetting samples... only subset on present variants."
+    )
```

Comment:
Good call out!  I was generally trying to mimic the behavior of the existing `load_data`, which used a `skip_sample_subset` param.  But `load_data` was fetching the sample ids from GCS rather than using the path as an input param.  I will fix!